reference/docs-conceptual/discover-powershell.md

```diff
@@ -77,197 +74,183 @@ standardized list of verbs. To get a full list of all possible verbs, use the `G
-That list you get back, is quite large though. To make finding commands easier you can limit the
-amount of information that comes back. You can filter the response using either parameters or by
-using helper cmdlets.
+The `Get-Command` cmdlet returns a list of all available commands installed on your system. The list
+you get back is quite large. You can limit the amount of information that comes back by filtering
+the response using parameters or helper cmdlets.
 
 ### Filter on name
 
-You can filter the output of `Get-Command`, using different parameters. Filtering this way, is about
-querying a specific property on the command. The idea is that you specify what property you want to
-filter against, and then you provide a string that you want to match against. You'll therefore get a
-comparison that looks like this:
+You can filter the output of `Get-Command` using different parameters. Filtering allows you to find
+commands that have certain properties. The **Name** parameter allows you to find a specific command
+by name.
 
 ```powershell
-Get-Command -Name '*Process'
+Get-Command -Name Get-Process
 ```
 
-At this point, the filtering is trying to do an exact matching against the provided string argument.
-If you want to have more flexibility, in the comparison, you can use a wildcard `*`, that does
-pattern matching. The following code would look for all commands, who's name ends with process:
+```Output
+CommandType     Name              Version    Source
+-----------     ----              -------    ------
+Cmdlet          Get-Process       7.0.0.0    Microsoft.PowerShell.Management
+```
 
-Above, the parameter the `-Name` is used to filter. Apart from `-Name`, you can also filter on
-`-ParameterName` and `-Type`, for example.
+What if you wanted to find all of the commands the work with processes. You can use a wildcard `*`
+to match other forms of the string. For example:
+
+```powershell
+Get-Command -Name *-Process
+```
+
+```Output
+CommandType     Name              Version    Source
+-----------     ----              -------    ------
+Cmdlet          Debug-Process     7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Get-Process       7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Start-Process     7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Stop-Process      7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Wait-Process      7.0.0.0    Microsoft.PowerShell.Management
+```
 
 ### Filtering on Noun and Verb
 
-You've seen how you can filter on `-Name`, and that there are other parameters you can filter on as
-well. Verb and noun is something you can filter on as well. Such filtering targets part of a
-command's name.
+There are other parameters that filter on verb and noun values. The verb part of a command's name is
+the leftmost part. The verb should be one of the values returned by the `Get-Verb` cmdlet. The
+rightmost part of a command is the noun part. A noun can be anything.
```

Comment:
Nit: Is it worth calling out that the noun should always be singular, not plural?

Reply:
Not here.